lib/inspec/objects/control.rb

```diff
@@ -18,13 +19,27 @@ def add_tag(t)
     end
 
     def to_hash
-      { id: id, title: title, desc: desc, impact: impact, tests: tests.map(&:to_hash), tags: tags.map(&:to_hash) }
+      {
+        id: id,
+        title: title,
+        descriptions: descriptions,
+        impact: impact,
+        tests: tests.map(&:to_hash),
+        tags: tags.map(&:to_hash),
+      }
     end
 
     def to_ruby # rubocop:disable Metrics/AbcSize
       res = ["control #{id.inspect} do"]
       res.push "  title #{title.inspect}" unless title.to_s.empty?
-      res.push "  desc  #{prettyprint_text(desc, 2)}" unless desc.to_s.empty?
+      descriptions&.each do |k, v|
```

Comment:
It's kinder to your fellow developers to use meaningful variable names.  Are the keys here labels?  Are the values text chunks?  Use word that reflect that.  Thanks!